selfdrive/controls/lib/latcontrol_torque.py

```diff
@@ -49,22 +46,27 @@ def update(self, active, CS, CP, VM, params, last_actuators, desired_curvature,
         actual_curvature = -VM.calc_curvature(math.radians(CS.steeringAngleDeg - params.angleOffsetDeg), CS.vEgo, params.roll)
       else:
         actual_curvature = llk.angularVelocityCalibrated.value[2] / CS.vEgo
-      desired_lateral_accel = desired_curvature * CS.vEgo**2
-      desired_lateral_jerk = desired_curvature_rate * CS.vEgo**2
-      actual_lateral_accel = actual_curvature * CS.vEgo**2
+
+      desired_lateral_accel = desired_curvature * CS.vEgo ** 2
+      desired_lateral_jerk = desired_curvature_rate * CS.vEgo ** 2
+      actual_lateral_accel = actual_curvature * CS.vEgo ** 2
 
       setpoint = desired_lateral_accel + LOW_SPEED_FACTOR * desired_curvature
       measurement = actual_lateral_accel + LOW_SPEED_FACTOR * actual_curvature
       error = setpoint - measurement
       pid_log.error = error
 
+      friction_compensation = interp(desired_lateral_jerk, [-JERK_THRESHOLD, JERK_THRESHOLD], [-self.friction, self.friction])
+      # avoid deadzone near saturation
+      self.pid.pos_limit = self.steer_max - friction_compensation
+      self.pid.neg_limit = -self.steer_max - friction_compensation
```

Comment:
This isn't needed if we think the impact of the deadzone at full torque isn't substantial